src/Dfe.PlanTech.Web/Views/Recommendations/RecommendationContent.cshtml

```diff
@@ -21,7 +23,16 @@
                             await Html.RenderPartialAsync("Components/PageComponentFactory", content);
                         }
                     </div>
+                    @if (headerWithContent is RecommendationChunk chunk && !string.IsNullOrEmpty(chunk.CSUrl))
```

Comment:
I reckon it's worth also checking that chunk.CSLinkText is not null or empty, if theres a link it certainly shouldn't be, but it is technically possible to have no link text in contentful when a link is there

Reply:
[2ada7b4](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/718/commits/2ada7b44a76a3d4bf6ddb3b76232fe56c42ece12)